packages/middleware-sdk-s3/src/s3-express/middleware-s3-express.e2e.spec.ts

```diff
@@ -261,52 +240,34 @@ async function createClientAndRecorder() {
   };
 }
 
-async function deleteBuckets(s3: S3) {
-  const buckets = await s3.listDirectoryBuckets({});
-
-  for (const bucket of buckets.Buckets ?? []) {
-    const Bucket = bucket.Name;
+async function deleteBuckets(s3: S3, bucketName: string) {
```

Comment:
rename function to `emptyAndDeleteBucket` singular?

Reply:
Done in [`99d92a2` (#6077)](https://github.com/aws/aws-sdk-js-v3/pull/6077/commits/99d92a26f3f36db1396f4253c30d07935d4ac92d)